sys/include/net/netopt.h

```diff
@@ -269,6 +269,54 @@ typedef enum {
      */
     NETOPT_LAST_ED_LEVEL,
 
+    /** @brief   Radio modulation bandwidth
+     *
+     * Depends on the type of radio used and the regional regulatory and is
+     * used with LoRa/FSK modems.
```

Comment:
> Which radio expect which type?

For the moment, only LoRa/FSK modem implement this but others could use it. I'm not enough expert with 802.15.4 PHY and implementations so I can't say.

> how is the top-layer (e.g. a MAC layer) supposed to choose one?

This is part of the LoRaMAC specifications (not done in this PR)